src/mixins/object_geometry.mixin.js

```diff
@@ -639,7 +639,7 @@
      * @return {Object} .y height dimension
      */
     _getNonTransformedDimensions: function() {
-      var strokeWidth = this.stroke ? this.strokeWidth : 0,
+      (this.stroke && this.stroke !== 'transparent') ? this.strokeWidth : 0,
           w = this.width + strokeWidth,
           h = this.height + strokeWidth;
       return { x: w, y: h };
```

Comment:
Missing `strokeWidth` here too

Reply:
Looks like I accidentally deleted it when fixing the formatting. Fixed now.